stress/src/throughput.rs

```diff
@@ -28,7 +29,28 @@ where
         STOP.store(true, Ordering::SeqCst);
     })
     .expect("Error setting Ctrl-C handler");
-    let num_threads = num_cpus::get();
+
+    let mut num_threads = num_cpus::get();
+    let mut args_iter = env::args();
+
+    if let Some(arg_str) = args_iter.nth(1) {
+        let arg = arg_str.parse::<usize>();
+
+        if !arg.is_ok() {
+            eprintln!("Invalid command line argument '{}' as number of threads. Make sure the value is a positive integer.", arg_str);
+            std::process::exit(1);
```

Comment:
I think we should use `Result.expect` instead of calling `process.exit`.

Something like this:

```rust
if let Some(arg_str) = args_iter.nth(1) {
    let arg = arg_str.parse::<usize>().expect("Provide a positive value for number of threads");
}

Reply:
I think in this case the slightly more verbose method is ok, as it provides more user-friendly error message.